tools/alpine/versions

```diff
@@ -0,0 +1,162 @@
+alpine-baselayout-3.0.4-r0
```

Comment:
I agree that having a list of exact versions if good to have. A note at the top would be sufficient.